.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Tests
```

Comment:
This Github Action will
- build docker image with functions loaded into emulator
- start a container with emulator
- run unit tests with `jest`
- run api integration tests with `newman`, the cli tool for `postman`